stubs/fpdf2/fpdf/encryption.pyi

```diff
@@ -64,29 +64,43 @@ class StandardSecurityHandler:
     o: str
     k: str
     u: str
+    # The following fields are only defined after a call to generate_user_password_rev6().
+    ue: Incomplete
+    # The following fields are only defined after a call to generate_owner_password_rev6().
+    oe: Incomplete
+    # The following fields are only defined after a call to generate_perms_rev6().
+    perms_rev6: Incomplete
 
     def __init__(
         self,
         fpdf: FPDF,
         owner_password: str,
         user_password: str | None = None,
-        permission: Incomplete | None = None,
-        encryption_method: EncryptionMethod | None = None,
+        permission: AccessPermission = ...,
+        encryption_method: EncryptionMethod = ...,
         encrypt_metadata: bool = False,
     ) -> None: ...
-    def generate_passwords(self, file_id) -> None: ...
+    def generate_passwords(self, file_id: str) -> None: ...
     def get_encryption_obj(self) -> EncryptionDictionary: ...
-    def encrypt(self, text: str | bytes | bytearray, obj_id) -> bytes: ...
-    def encrypt_string(self, string, obj_id): ...
-    def encrypt_stream(self, stream, obj_id): ...
+    def encrypt(self, text: str | bytes | bytearray, obj_id: int) -> str | bytes: ...
```

Comment:
This should use an overload, otherwise users have to use an `isinstance` check when providing either a `str` or a `bytes` (but not both):

```suggestion
    @overload
    def encrypt(self, text: bytes | bytearray, obj_id: int) -> bytes: ...
    @overload
    def encrypt(self, text: str, obj_id: int) -> str: ...
```

(`overload`  needs to be added to the imports.)